tools/toc.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Run `markdown-toc` on all Markdown files with tables of contents
+# to see if any of them need an updated TOC.
+#
+# As a side effect, if you run this locally it will generate
+# all needed TOC and update the markdown documents.
+
+let ERROR=0
+
+# Find all markdown files
+for f in $(find . | grep .md); do
+
+	# Only use ones that include a table of contents
+	grep '<!-- tocstop -->' $f > /dev/null
```

Comment:
I think you should look for `<!-- toc -->` instead, that way this script will also work/update files that add a new table of contents